### DISCUSSION TITLE
Support Custom Homepage Locally

### DISCUSSION THREAD

**Discussion by ghost** [colinwilson/lotusdocs#21] [Feature Requests]
Building the theme locally using the [quickstart guide](https://lotusdocs.dev/docs/quickstart/) pulls the homepage from [lotusdocs.dev](https://lotusdocs.dev/).

A default content layout (eg `class="docs-content col-12 col-xl-9 mt-0"`) from a subsection like `docs/overview/`, `docs/quickstart/`, etc. on the homepage would be really great.

**Comment by colinwilson** [comment]
Do you mean have the `/docs` path as the site's root/ home page?

**Reply by ghost** [reply]
Apologies, I didn't explain well.

I meant for the homepage to have the width of any page in `/docs` **without** the menu on the left and ToC on the right.

If I comment out the `image:` block in [this file](https://github.com/colinwilson/lotusdocs/blob/release/data/landing.yaml#L20), the text does not span as wide as [this section](https://lotusdocs.dev/docs/overview/#what-is-lotus-docs) for instance.

It makes sense for the homepage to have lesser width than a content page, but if the `image:` block is commented, the text should definitely span wider.

**Reply by colinwilson** [reply]
![2023-08-17 16_02_38-landing yaml - lotusdocs dev - Visual Studio Code](https://github.com/colinwilson/lotusdocs/assets/12916656/17a5f94c-7fde-4463-84e4-9554ff6b0138)

Oh, so have the **Lotus Docs** and content below span to the right, right?

**Reply by ghost** [reply]
Yes !

**Reply by colinwilson** [reply]
Gotcha. I'll update the hero template to fill the page when the `image:` block isn't declared.

P.S. I have alternative hero templates in the works 😉.
